math/dlib/Portfile

```diff
@@ -33,3 +38,50 @@ compiler.blacklist-append {clang < 800}
 
 configure.cxxflags-append -fvisibility=hidden -fvisibility-inlines-hidden
 cmake.build_type    Release
+
+# create Python subports
+set python_versions {37 38}
+foreach v ${python_versions} {
+    subport py${v}-${github.project} {
+        PortGroup   python 1.0
+        python.default_version  ${v}
+    }
+}
+
+# Python bindings for supported Python versions
+if {[string match "py*" ${subport}]} {
+    github.setup    ${github.author} ${github.project} ${github.version} ${github.tag_prefix}
+
+    categories-append   math
+    description     Python ${python.branch} bindings for ${description}
+    long_description\
+                    This package provides Python ${python.branch}\
+                    bindings, modules, and scripts.
+
+    depends_lib-append \
+                    port:sqlite3
+
+    depends_build-append \
+                    port:cmake \
```

Comment:
Please allow cmake-devel to satisfy this dependency by writing `path:bin/cmake:cmake` instead of `port:cmake`. But the cmake portgroup that the port already includes should already be adding this dependency. Are you sure you need to add it manually?

Reply:
Fixed thanks.